<ui version="4.0" >
 <class>BatchTMDialog</class>
 <widget class="QDialog" name="BatchTMDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Batch Tone Mapping</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >
    <normaloff>:/new/prefix1/images/qtpfsgui.png</normaloff>:/new/prefix1/images/qtpfsgui.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="margin" >
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>9</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Input Hdr list</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>5</number>
        </property>
        <property name="margin" >
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="add_dir_HDRs_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Add all the HDRs in a directory to the list</string>
            </property>
            <property name="text" >
             <string>Add &amp;Directory</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/fileopen.png</normaloff>:/new/prefix1/images/fileopen.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="add_HDRs_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Add single HDR files to the list</string>
            </property>
            <property name="text" >
             <string>&amp;Add Files</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/add.png</normaloff>:/new/prefix1/images/add.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remove_HDRs_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Remove single HDR files to the list</string>
            </property>
            <property name="text" >
             <string>&amp;Remove Files</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/remove.png</normaloff>:/new/prefix1/images/remove.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_HDRs" >
          <property name="toolTip" >
           <string>List of HDRs that will be tone mapped</string>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5" >
       <property name="title" >
        <string>Input Tone Mapping Setting list</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>5</number>
        </property>
        <property name="margin" >
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="add_dir_TMopts_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Add all the Tone Mapping Setting files in a directory to the list</string>
            </property>
            <property name="text" >
             <string>Add D&amp;irectory</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/fileopen.png</normaloff>:/new/prefix1/images/fileopen.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="add_TMopts_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Add single Tone Mapping Setting files to the list</string>
            </property>
            <property name="text" >
             <string>Add Fi&amp;les</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/add.png</normaloff>:/new/prefix1/images/add.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remove_TMOpts_Button" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Remove single Tone Mapping Setting files to the list</string>
            </property>
            <property name="text" >
             <string>R&amp;emove Files</string>
            </property>
            <property name="icon" >
             <iconset resource="../icons.qrc" >
              <normaloff>:/new/prefix1/images/remove.png</normaloff>:/new/prefix1/images/remove.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_TMopts" >
          <property name="toolTip" >
           <string>List of Tone Mapping Setting files that will be used to tone map each HDR</string>
          </property>
          <property name="selectionMode" >
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="toolTip" >
      <string>Specify output folder for the tone mapped files</string>
     </property>
     <property name="title" >
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="margin" >
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="out_folder_Button" >
        <property name="text" >
         <string>Select &amp;Output Folder...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="out_folder_widgets" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Log:</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="margin" >
       <number>2</number>
      </property>
      <item>
       <widget class="QListView" name="Log_Widget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Batch operations report</string>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="resizeMode" >
         <enum>QListView::Adjust</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="filterLabel1" >
          <property name="text" >
           <string>&amp;Show only:</string>
          </property>
          <property name="buddy" >
           <cstring>filterComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterComboBox" >
          <property name="toolTip" >
           <string>Filter messages based on severity</string>
          </property>
          <item>
           <property name="text" >
            <string>All messages</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Errors only</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Success messages</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filterLabel2" >
          <property name="text" >
           <string>&amp;Filter Log messages:</string>
          </property>
          <property name="buddy" >
           <cstring>filterLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filterLineEdit" />
        </item>
        <item>
         <widget class="QToolButton" name="clearTextToolButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Clear filter text</string>
          </property>
          <property name="icon" >
           <iconset resource="../icons.qrc" >
            <normaloff>:/new/prefix1/images/fileclose.png</normaloff>:/new/prefix1/images/fileclose.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="overallProgressBar" >
     <property name="toolTip" >
      <string>Overall completion progress</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelbutton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BatchGoButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Start batch tone mapping</string>
       </property>
       <property name="text" >
        <string>&amp;Start!</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>add_dir_HDRs_Button</tabstop>
  <tabstop>add_HDRs_Button</tabstop>
  <tabstop>remove_HDRs_Button</tabstop>
  <tabstop>add_dir_TMopts_Button</tabstop>
  <tabstop>add_TMopts_Button</tabstop>
  <tabstop>remove_TMOpts_Button</tabstop>
  <tabstop>out_folder_Button</tabstop>
  <tabstop>filterComboBox</tabstop>
  <tabstop>cancelbutton</tabstop>
  <tabstop>BatchGoButton</tabstop>
  <tabstop>listWidget_HDRs</tabstop>
  <tabstop>listWidget_TMopts</tabstop>
  <tabstop>out_folder_widgets</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelbutton</sender>
   <signal>clicked()</signal>
   <receiver>BatchTMDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>597</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel" >
     <x>327</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearTextToolButton</sender>
   <signal>clicked()</signal>
   <receiver>filterLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>660</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel" >
     <x>611</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
